R/textmodel_nb.R

```diff
@@ -331,3 +331,80 @@ colNorm <- function(x) {
 print.predict.textmodel_nb <- function(x, ...) {
     print(unclass(x))
 }
+
+
+#' Compute the information gain of features from a fitted \link{textmodel_nb}
+#' object
+#' 
+#' Computes the information gain of features from a fitted \link{textmodel_nb}
+#' object.  See  
```

Comment:
add citation here

Reply:
@kbenoit I agree it would be useulf to cross-check the computation with a well-known example. Jurafsky et al. do not provide one. maybe looking on google scholar can help?

Reply:
Sure, anything we/you can find would be useful (and citeable).